.github/workflows/release.yml

```diff
@@ -0,0 +1,60 @@
+# release.yml
+# bump a new version, create a new tag, and create a release in Github Releases
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Input a new version number to release. (e.g. 1.2.3)'
+        required: true
+
+run-name: Release v${{github.event.inputs.version}}
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          cache: "yarn"
+
+      - name: Bump version
+        run: |
+          yarn install
+          yarn run update-version ${{github.event.inputs.version}}
+          yarn run update-changelog ${{github.event.inputs.version}}
+          git config --global user.name 'github-actions[bot]'
+          git config --global user.email 'github-actions[bot]@users.noreply.github.com'
+          git add .
+          git commit -m "chore(release): v${{github.event.inputs.version}}"
+          git tag -a v${{github.event.inputs.version}} -m "Release v${{github.event.inputs.version}}"
+          git remote set-url origin https://x-access-token:${GITHUB_TOKEN}@github.com/$GITHUB_REPOSITORY
+          git push origin
```

Comment:
Hi, @wxharry, with my understanding, `yarn run update-version` won't check if the new version is greater than the old one, right? Do you intend to make use of `git tag` to output a possible error? Since there should not be two same tags in a git repository.

Anyway, whatever can make sure a new version is always greater than the old would be OK.

Reply:
Well, we do need to compare versions. `git tag` alone cannot guarantee that a smaller version number will stop the release workflow and flag it `fail`.

Suppose the released tags are as below:

- v1.8.4
- v1.8.0
- v1.7.3
- vx.x.x

Now if we carelessly input a tag `v1.8.3` which is smaller than `v1.8.4` however it can be released in the end. Though finally in the deploy process, the two stores will both reject them due to a bad version number, a release is already sucessfully but unexpectedly generated in our repo.

Reply:
Oops, an [error](https://github.com/tyn1998/workflows-for-hypercrx/actions/runs/3946374592/jobs/6754082677) when I tested the release workflow:

![image](https://user-images.githubusercontent.com/32434520/213102741-2ce25716-1221-42f0-9592-d5e4229337be.png)

It seems to be related to arguments. I'll look into it and try to figure it out myself.

Reply:
> Well, we do need to compare versions. `git tag` alone cannot guarantee that a smaller version number will stop the release workflow and flag it `fail`.
Sure. I will add a new function to check if the version number is greater. And fail the workflow if it isn't.

Reply:
Tested and confirmed! Great job!

- https://github.com/tyn1998/workflows-for-hypercrx/actions/runs/4021452583/jobs/6910332818
- https://github.com/tyn1998/workflows-for-hypercrx/actions/runs/4021460030/jobs/6910348377
- https://github.com/tyn1998/workflows-for-hypercrx/actions/runs/4021464433/jobs/6910357841